certora/applyHarness.patch

```diff
@@ -42,20 +42,84 @@ diff -druN Safe.sol Safe.sol
      }
  }
 diff -druN base/Executor.sol base/Executor.sol
---- base/Executor.sol	2023-11-10 09:50:31
-+++ base/Executor.sol	2023-11-20 10:28:47
-@@ -26,12 +26,8 @@
-         uint256 txGas
-     ) internal returns (bool success) {
-         if (operation == Enum.Operation.DelegateCall) {
--            /* solhint-disable no-inline-assembly */
--            /// @solidity memory-safe-assembly
--            assembly {
+--- base/Executor.sol	2023-12-06 16:00:31
++++ base/Executor.sol	2023-12-07 00:01:59
+@@ -29,8 +29,10 @@
+             /* solhint-disable no-inline-assembly */
+             /// @solidity memory-safe-assembly
+             assembly {
 -                success := delegatecall(txGas, to, add(data, 0x20), mload(data), 0, 0)
--            }
--            /* solhint-enable no-inline-assembly */
-+            // MUNGED lets just be a bit more optimistic, `execute` does nothing for `DELEGATECALL` and always returns true
++                // success := delegatecall(txGas, to, add(data, 0x20), mload(data), 0, 0)
++                // MUNGED: lets be optimistic and assume execute does nothing for DELEGATECALL
+             }
 +            return true;
+             /* solhint-enable no-inline-assembly */
          } else {
              /* solhint-disable no-inline-assembly */
-             /// @solidity memory-safe-assembly
+diff -druN base/Executor.sol.orig base/Executor.sol.orig
+--- base/Executor.sol.orig	1970-01-01 05:30:00
++++ base/Executor.sol.orig	2023-12-07 00:01:59
```

Comment:
This section of the diff adds a new `.orig` file, is this intended? To me, this looks like local changes that made it to the patch.

Reply:
Rectified 👍🏾